PayPal/src/test/java/com/braintreepayments/api/PayPalPendingRequestUnitTest.kt

```diff
@@ -0,0 +1,64 @@
+package com.braintreepayments.api
+
+import android.net.Uri
+import org.json.JSONException
+import org.json.JSONObject
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
```

Comment:
Is Robolectric needed for this unit test? Do we use Robolectric for all of our unit tests?

Reply:
We use it in most but not all unit tests for mocking Android dependencies. In these specific tests we get an error that `parse` in `Uri` is not mocked without Robolectric. We use the `AndroidJUnitRunner` for some tests where Robolectric is not required. Definitely open to improvements in our test suite, Robolectric has just been the default for a while now.